src/test/java/org/openrewrite/launchdarkly/UpgradeLaunchDarkly6Test.java

```diff
@@ -92,44 +94,154 @@ void mavenDependency() {
+                  }
+                  """
+              )
+            );
+        }
+
+        @Test
         void userToContext() {
             rewriteRun(
               //language=java
               java(
                 """
+                  import com.launchdarkly.sdk.LDUser;
+                  
                   class A {
                     void foo() {
-                      LDUser user = new LDUser.Builder("user-key-123abc")
-                        .name("Sandy")
-                        .email("sandy@example.com")
-                        .custom("groups",
-                          LDValue.buildArray().add("Google").add("Microsoft").build())
-                        .build();
+                      LDUser user = new LDUser("user-key-123abc");
                     }
                   }
                   """,
                 """
+                  import com.launchdarkly.sdk.LDContext;
+                  
                   class A {
                     void foo() {
-                      LDContext context = LDContext.builder("user-key-123abc")
-                        .name("Sandy")
-                        .set("email", "sandy@example.com")
-                         .set("groups",
-                          LDValue.buildArray().add("Google").add("Microsoft").build())
-                        .build();
+                      LDContext user = LDContext.create("user-key-123abc");
                     }
                   }
                   """
               )
             );
         }
+
+        @Test
+        void privateAttribute() {
+            rewriteRun(
+              //language=java
+              java(
+                """
+                  import com.launchdarkly.sdk.LDUser;
+                  import com.launchdarkly.sdk.LDValue;
+                  
+                  class A {
+                      void foo() {
+                          LDUser user = new LDUser.Builder("user-key-123abc")
+                                  .name("Sandy")
+                                  .privateEmail("sandy@example.com")
+                                  .privateCustom("groups", LDValue.buildArray().add("Google").add("Microsoft").build())
+                                  .build();
+                      }
+                  }
+                  """,
+                """
+                  import com.launchdarkly.sdk.LDContext;
+                  import com.launchdarkly.sdk.LDValue;
+                  
+                  class A {
+                      void foo() {
+                          LDContext user = LDContext.builder("user-key-123abc")
+                                  .name("Sandy")
+                                  .set("email", "sandy@example.com")
+                                  .privateAttributes("email")
+                                  .set("groups", LDValue.buildArray().add("Google").add("Microsoft").build())
+                                  .privateAttributes("groups")
```

Comment:
The `privateAttributes(java.lang.String...)` accepts varargs, but I didn't spot an easy way without possibly using a secondary visitor to add the private attribute method with all of the arguments at once.

Ideal after:
```java
import com.launchdarkly.sdk.LDContext;
import com.launchdarkly.sdk.LDValue;
                  
class A {
    void foo() {
        LDContext user = LDContext.builder("user-key-123abc")
                .name("Sandy")
                .set("email", "sandy@example.com")
                .set("groups", LDValue.buildArray().add("Google").add("Microsoft").build())
                .privateAttributes("email", "groups")
                .build();
    }
}                                 
```